docs/docs/01-ibc/05-upgrades/01-quick-guide.md

```diff
@@ -30,7 +35,7 @@ Note: Since upgrades are only implemented for Tendermint clients, this doc only
 
 If the IBC-connected chain is conducting an upgrade that will break counterparty clients, it must ensure that the upgrade is first supported by IBC using the list above and then execute the upgrade process described below in order to prevent counterparty clients from breaking.
 
-1. Create a 02-client [`UpgradeProposal`](https://github.com/cosmos/ibc-go/blob/main/docs/ibc/proto-docs.md#upgradeproposal) with an `UpgradePlan` and a new IBC ClientState in the `UpgradedClientState` field. Note that the `UpgradePlan` must specify an upgrade height **only** (no upgrade time), and the `ClientState` should only include the fields common to all valid clients and zero out any client-customizable fields (such as TrustingPeriod).
+1. Create a 02-client [`UpgradeProposal`](../08-proto-docs.md)with an `UpgradePlan` and a new IBC ClientState in the `UpgradedClientState` field. Note that the `UpgradePlan` must specify an upgrade height **only** (no upgrade time), and the `ClientState` should only include the fields common to all valid clients and zero out any client-customizable fields (such as TrustingPeriod).
```

Comment:
Correct. I tried to stay away from content updates in this PR.

Also, I think we may wanna build our buf docs using docusaurus in the future as there is support for this, so these links can become useful one day

Reply:
Is there an issue to track fixing this? Seems a bit confusing to leave a link if it isn't particularly useful. I agree it can be fixed later, but would just like to make sure we don't lose track of this